src/test/key_tests.cpp

```diff
@@ -344,4 +345,33 @@ BOOST_AUTO_TEST_CASE(bip340_test_vectors)
     }
 }
 
+BOOST_AUTO_TEST_CASE(bip324_ecdh)
+{
+    CKey initiator_key = DecodeSecret(strSecret1);
+    CKey responder_key = DecodeSecret(strSecret2C);
+
+    ECDHSecret initiator_secret, responder_secret;
+    auto initiator_hdata_vec = ParseHex("2deb41da6887640dda029ae41c9c9958881d0bb8e28f6bb9039ee9b7bb11091d62f4cbe65cc418df7aefd738f4d3e96c66365b4d38eefd0a883be64112f4495");
```

Comment:
to match the [64 bytes encoding](https://github.com/bitcoin/bitcoin/blob/c9db50bc2f8674913e785384c0e323c2df71754d/src/test/net_tests.cpp#L926) in commit c9db50b.
```suggestion
    auto initiator_hdata_vec = ParseHex("2deb41da6887640dda029ae41c9c9958881d0bb8e28f6bb9039ee9b7bb11091d62f4cbe65cc418df7aefd738f4d3e926c66365b4d38eefd0a883be64112f4495");
```

Reply:
Thanks! Done.